### ISSUE TITLE
Termbox2 dependency issue

### ISSUE THREAD

**Issue by rmorenohernandez** [adsr/phpspy#131]
Hi, 

I've detected an issue when launching make. I checked out that issue is due to project dependency termbox2, that changed on 2nd April (the change that impact is the renamed file termbox.h -> termbox2.h ,  https://github.com/termbox/termbox2/commit/a4f92ea894da9989c9ce5ec998e91fec9cdd4995 ). It is necessary to change phpspy.c and phpspy.h to include  termbox2.h. instead of  termbox.h

**Comment by adsr** [comment]
Hello. The termbox2 submodule is pinned at c68d9b0 which has `termbox.h`. Next time I bump the submodule I'll rename. Given that it's a single file dependency, I may copy the file here instead of submoduling it, similar to uthash.

**Comment by senseicoder** [comment]
Hello, same problem here, just after git clone, on a fresh working copy, the pin is not work (GNU Make 4.1, git 2.17.1) : 
```
cedric@Mnementh6 ~/www/e/phpspy (master) $ make
git submodule update --init --remote --recursive
Chemin de sous-module 'vendor/termbox2' : '915b186bdc94c6a79537dbe1e7479af2d26f5081' extrait
cd vendor/termbox2 && git reset --hard
HEAD est maintenant à 915b186 invalidate last_(fg|bg) after mode switch
cc -std=c11 -Wall -Wextra -pedantic -g -O3 -Wno-address-of-packed-member  -I. -I./vendor -Ivendor/termbox2/  phpspy.c pgrep.c top.c addr_objdump.c event_fout.c event_callgrind.c -o phpspy  -pthread 
In file included from phpspy.c:1:0:
phpspy.h:30:10: fatal error: termbox.h: Aucun fichier ou dossier de ce type
 #include <termbox.h>
          ^~~~~~~~~~~
compilation terminated.
In file included from pgrep.c:1:0:
phpspy.h:30:10: fatal error: termbox.h: Aucun fichier ou dossier de ce type
 #include <termbox.h>
          ^~~~~~~~~~~
compilation terminated.
In file included from top.c:1:0:
phpspy.h:30:10: fatal error: termbox.h: Aucun fichier ou dossier de ce type
 #include <termbox.h>
          ^~~~~~~~~~~
compilation terminated.
In file included from addr_objdump.c:1:0:
phpspy.h:30:10: fatal error: termbox.h: Aucun fichier ou dossier de ce type
 #include <termbox.h>
          ^~~~~~~~~~~
compilation terminated.
In file included from event_fout.c:1:0:
phpspy.h:30:10: fatal error: termbox.h: Aucun fichier ou dossier de ce type
 #include <termbox.h>
          ^~~~~~~~~~~
compilation terminated.
In file included from event_callgrind.c:1:0:
phpspy.h:30:10: fatal error: termbox.h: Aucun fichier ou dossier de ce type
 #include <termbox.h>
          ^~~~~~~~~~~
compilation terminated.
Makefile:32: recipe for target 'phpspy' failed
make: *** [phpspy] Error 1
```
renaming termbox.h in phpsyp.[ch] works.

**Comment by adsr** [comment]
The submodule is pinned at c68d9b0 but looks like you're getting 915b186 due to the `--remote` flag. In any case I will update this now.

EDIT: Sorry I just saw the `Makefile` includes the `--remote` flag. I will fix that too.

**Comment by adsr** [comment]
Fixed in aff5b02c96c438ce53f2431a0e0b8c8f5e0dbdb5. Thank you for reporting.
